From 54ac159d360a8fc9e8fcb19d984e8230e16f6fc0 Mon Sep 17 00:00:00 2001
From: Syed Hasan Raza Naqvi <syed.naqvi@broadcom.com>
Date: Wed, 4 May 2022 05:11:01 +0000
Subject: [PATCH] From dfe0158344ea581370dd2dba8e4db189cf369147 Mon Sep 17
 00:00:00 2001 Subject: [PATCH] Disable ipv6 src address test in pcep. Docker
 lo interface  doesn't have ipv6 enabled. Socket bind returns cannot assign
 requested  address.


diff --git a/pceplib/test/pcep_socket_comm_test.c b/pceplib/test/pcep_socket_comm_test.c
index 116531f12..8b71a5090 100644
--- a/pceplib/test/pcep_socket_comm_test.c
+++ b/pceplib/test/pcep_socket_comm_test.c
@@ -92,7 +92,7 @@ void pcep_socket_comm_test_setup()
 	inet_pton(AF_INET, "127.0.0.1", &(test_host_ip));
 	inet_pton(AF_INET, "127.0.0.1", &(test_src_ip));
 	inet_pton(AF_INET6, "::1", &(test_host_ipv6));
-	inet_pton(AF_INET6, "::1", &(test_src_ipv6));
+	inet_pton(AF_INET6, "::", &(test_src_ipv6));
 }
 
 void pcep_socket_comm_test_teardown()
-- 
2.17.1

